src/_P164_gases_ens160.ino

```diff
@@ -41,7 +41,7 @@ boolean Plugin_164(uint8_t function, struct EventStruct *event, String& string)
       Device[deviceCount].TimerOption        = true;
       Device[deviceCount].GlobalSyncOption   = true;
       Device[deviceCount].PluginStats        = true;
-      Device[deviceCount].I2CNoDeviceCheck   = true;  //TODO
+      Device[deviceCount].I2CNoDeviceCheck   = true;
```

Comment:
That device flag should _not_ be needed, unless the device has a very unreliable I2C behavior. This flag _disables_ the regular I2C 'alive' check. You could comment this line, have the global I2C check (Tools/Advanced) enabled, and enable the task. If after some running time (hours), the task is still enabled and running, you don't need this flag.

Reply:
Flag was a bit unclear and copied from another I2C device. Good to know flag should not be set for standard I2C devices. Revoved.